### PR TITLE
Appearances

### PR DESCRIPTION
**Pull request by felix-horn**
Please review my "Appearances" feature.

### CODE REVIEW

**Review by benjamintautfest — APPROVED** [review]
seems reasonable to me :-)

File: src/CharacterCard.js [hunk 1]
```diff
@@ -11,10 +13,56 @@ export default function CharacterCard({ character, resetCard }) {
...
+
+  const episodeNumbersArray = episode.map((singleEpisode) =>
+    singleEpisode.slice(singleEpisode.lastIndexOf('/') + 1)
+  )
+
+  const htmlSnipetEpisodes = `episode/${episodeNumbersArray.join(',')}`
+
+  useEffect(() => {
+    getData(htmlSnipetEpisodes).then((data) => {
+      data = Array.isArray(data) ? data : [data]
```
Comment (mmurakaru) [inline src/CharacterCard.js:29]: Please explain this line :)

File: src/CharacterCard.js [hunk 2]
```diff
@@ -105,11 +157,55 @@ const SectionStyled = styled.section`
...
     background: none;
     border: none;
     color: #4a4a4a;
   }
 `
+const BackgroundBlur = styled.div`
+  background-color: #04292fee;
+  border-radius: 1px;
+  position: absolute;
+  z-index: 250;
```
Comment (mmurakaru) [inline src/CharacterCard.js:172]: z-index: 1 would be enough ;)

**Review by JanaHaeusler — COMMENTED** [review]
Great work! Thank you!
Would love to go through the code in order to fully grap the functionality in 1-2 places. :)

File: src/CharacterCard.js (hunk 1 above)
Comment (felix-horn) [reply src/CharacterCard.js:29]: This is the exact solution Torsten used and explained to us yestereday in the beginning - or maybe that was before you joined. Anyways :) If the character appeared in just one episode, the getData-function returns an object and not an array. This simple ternary operator turns the object into an array. And tadaa!

**Pull request merged**